<map id="triton::exceptions::PyCallbacks" name="triton::exceptions::PyCallbacks">
<area shape="rect" id="Node000001" title=" " alt="" coords="5,168,127,208"/>
<area shape="rect" id="Node000002" href="$classtriton_1_1exceptions_1_1Exception.html" title="The root class of all exceptions." alt="" coords="5,79,127,120"/>
<area shape="poly" id="edge1_Node000001_Node000002" title=" " alt="" coords="69,135,69,167,63,167,63,135"/>
<area shape="rect" id="Node000003" title=" " alt="" coords="14,5,118,31"/>
<area shape="poly" id="edge2_Node000002_Node000003" title=" " alt="" coords="69,46,69,79,63,79,63,46"/>
</map>
